Dockerfile.static

```diff
@@ -1,7 +1,7 @@
-FROM fedora:latest
+FROM docker.io/fedora:latest
```

Comment:
I would prefer
registry.fedoraproject.org/fedora:latest
docker.io is just a redirect to fedora in this case, I believe
Does this work for you?